### ISSUE TITLE
gopass segfaults when built with a pie-enabled go compiler

### ISSUE THREAD

**Issue by eli-schwartz** [gopasspw/gopass#710]
Arch Linux has been trying to move to universal PIE, and currently offers go as well as go-pie; we're trying to get into a state where everything can build with go-pie.

Attempting to build gopass using go-pie results in a binary that simply goes `Segmentation fault      (core dumped)`

I could get a working gopass binary built with go-pie by removing ~~~`CGO_ENABLED=0` and~~~ `-extldflags '-static'`, but TBH I have no idea why it is being built statically in the first place.

/cc @Foxboron

EDIT: correction, removing -static on its own is all that is needed to get things to work. I dunno what cgo is doing here or why it needs to be explicitly disabled, except it spawns a warning `loadinternal: cannot find runtime/cgo`.

**Comment by dominikschulz** [comment]
Thank you very much for raising this issue.

It took me some time to actually remember why we decided to do static builds in the first place.
Actually this is something we just adopted from our internal build scripts where we used to build Go binaries on Ubuntu and run them in Docker (`FROM scratch` or alpine). This didn't work with dynamically linked Go binaries as the required C lib were not available or not compatible.

For gopass this is not strictly necessary and we can discuss if we want to remove the `CGO_ENABLED=0` and `-extldflags=-static` flags. Right now I'm not entirely sure if this is worth the effort.
